recipes/leftbar/contact_row/contact_row.vue

```diff
@@ -14,28 +14,14 @@
   >
     <template #left>
       <dt-avatar
+        :full-name="name"
+        :image-src="avatarSrc"
+        :icon-name="noInitials ? 'user' : null"
```

Comment:
could we move this to a computed property so we will not use logic in template

Reply:
Also I see in some places we are using
`:icon-name="avatarIcon"` or `:icon-name="iconName"` it is probably because in some cases is taken by stories.js, could we take a look on this please?

Reply:
> Also I see in some places we are using `:icon-name="avatarIcon"` or `:icon-name="iconName"` it is probably because in some cases is taken by stories.js, could we take a look on this please?

What should I do here? I don't see any problem really

Reply:
k cool, nvm I thought maybe it's some like inconsistency 👍🏼